client/searchResults.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", getResults);
+
+function getResults() {
+    fetch("http://localhost:5000/pizzaresult")
+        .then((r) => r.text())
+        .then(
+            (result) =>
+                (document.getElementbyId("results").textContent = result)
+        );
+}
```

Comment:
after enabling cors on server you will be able to get the data but you will need to write a function to parse it in the DOM eg:

```document.addEventListener("DOMContentLoaded", getResults);

function getResults() {
    fetch("http://localhost:5000/pizzaresult")
        .then((r) => r.json())
        .then(
            (r) => {
               parseData(r)
            }
        ).catch(console.warn);
}

let parent = document.getElementById('results')
function parseData(data) {
    for (let i=0; i<data.length; i++) {
        let listItem = document.createElement('li')
        listItem.innerText = JSON.stringify(data)
        parent.appendChild(listItem)
    }
}